Files/UserControls/YourHome.xaml.cs

```diff
@@ -336,18 +234,46 @@ private void MenuFlyoutItem_Click(object sender, RoutedEventArgs e)
             Empty.Visibility = Visibility.Visible;
         }
 
-        private void DropShadowPanel_PointerPressed(object sender, Windows.UI.Xaml.Input.PointerRoutedEventArgs e)
+        private void Button_Click(object sender, RoutedEventArgs e)
         {
-            (sender as DropShadowPanel).ShadowOpacity = 0.025;
-        }
+            string NavigationPath = ""; // path to navigate
+            string ClickedCard = (sender as Button).Tag.ToString();
 
-        private void DropShadowPanel_PointerReleased(object sender, Windows.UI.Xaml.Input.PointerRoutedEventArgs e)
-        {
-            (sender as DropShadowPanel).ShadowOpacity = 0.25;
-        }
+            switch (ClickedCard)
+            {
+                case "Downloads":
+                    NavigationPath = App.AppSettings.DownloadsPath;
+                    break;
 
-        private void RecentsView_RightTapped(object sender, Windows.UI.Xaml.Input.RightTappedRoutedEventArgs e)
-        {
+                case "Documents":
+                    NavigationPath = App.AppSettings.DocumentsPath;
+                    break;
+
+                case "Pictures":
+                    NavigationPath = App.AppSettings.PicturesPath;
+                    break;
+
+                case "Music":
+                    NavigationPath = App.AppSettings.MusicPath;
+                    break;
+
+                case ("Videos"):
+                    NavigationPath = App.AppSettings.VideosPath;
+                    break;
+            }
+
+            switch (App.AppSettings.LayoutMode)
+            {
+                case 0:
+                    App.CurrentInstance.ContentFrame.Navigate(typeof(GenericFileBrowser), NavigationPath); // List View
+                    break;
+
+                case 1:
+                    App.CurrentInstance.ContentFrame.Navigate(typeof(PhotoAlbum), NavigationPath); // Grid View
```

Comment:
I think there will be a conflict here with the latest changes to layout modes.

Reply:
Good catch 😸🎖